.gitignore

```diff
@@ -0,0 +1,75 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
I just used the magickal templates provided by IntelliJ.  I suppose I could go hand-jam it...